api/v1/openapi.yaml

```diff
@@ -3447,3 +3463,58 @@ definitions:
         type: array
         items:
           type: string
+  BgpPeer:
+    description: |-
+      State of a BGP Peer
+      
+      +k8s:deepcopy-gen=true
+    properties:
+      local-asn:
+        description: Local AS Number
+        type: integer
+      peer-asn:
+        description: Peer AS Number
+        type: integer
+      peer-address:
+        description: IP Address of peer
+        type: string
+      session-state:
+        description: |
+          BGP peer operational state as described here
+          https://www.rfc-editor.org/rfc/rfc4271#section-8.2.2
+        type: string
+      admin-state:
+        description: BGP peer configured administrative state
+        type: string
+      uptime:
```

Comment:
I'd prefer to return an integer (with seconds let's say) instead of a formatted string from API. Also, I think it would be nice to have an explicit unit like `uptime-seconds`.

Reply:
Done, returning uptime in integer. In CLI, parsing it with time.Duration, so it can be human readable.

Reply:
Great! Can you put the unit to the field name as well? Like `uptime-seconds`.

Reply:
Updated field name to uptime-nanoseconds